src/Components/Pages/ConfirmAddress/ConfirmAddress.tsx

```diff
@@ -20,11 +25,23 @@ export const ConfirmAddress: React.FC = () => {
   const pitch = "0";
   const width = "814";
   const height = "356";
-  // const size = "634x352";
   const marker = `pin-s+000(${longLat})`;
 
   const mapImageURL = `https://api.mapbox.com/styles/v1/${styleToken}/static/${marker}/${longLat},${zoom},${bearing},${pitch}/${width}x${height}?access_token=${accessToken}`;
 
+  const handleSubmit = async () => {
+    try {
+      trigger({
+        id: user?.id,
+        address_confirmed: true,
+        nycdb_results: bldgData,
+      });
+    } catch (error) {
+      console.log({ "tenants2-error": error });
```

Comment:
should we log this error in rollbar?

Reply:
oh right yeah good point. I'll make a separate ticket to add all the rollbar stuff.